lib/handler/http2_debug_state.c

```diff
@@ -19,18 +19,17 @@
  * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
  * IN THE SOFTWARE.
  */
-#include "h2o.h"
-
 #include <inttypes.h>
+#include "h2o.h"
 
-struct st_h2o_debug_state_handler_t {
+struct st_h2o_http2_debug_state_handler_t {
     h2o_handler_t super;
     int hpack_enabled : 1;
```

Comment:
If you want a boolean, please use `unsigned : 1` that takes either 0 or 1. `int :1` takes two either 0 or -1.

Or simply using `int` or `unsigned` without `:nnn` would be sufficient in this case, considering the fact that the size of the struct would be indifferent and that we wouldn't have many instances of this struct.